zarf.schema.json

```diff
@@ -155,6 +155,12 @@
     },
     "ZarfComponentScripts": {
       "properties": {
+        "showOutput": {
+          "type": "boolean"
+        },
+        "timeout": {
```

Comment:
The Go variable is timeoutSeconds, which I like since it makes units clear. Since a good amount of apps also use milliseconds for their timeout, what do you think about adding 'seconds' to the key name to avoid confusion?

Reply:
Hmmm I guess we could, I did seconds because this will be to set the timeout for a long period (in most cases) and milliseconds seems a bit crazy for that.  But it doesn't destroy my soul to change it.